### PR TITLE
reactjs customizing lock

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero please review

File: articles/client-platforms/react/10-customizing-lock.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## 1. Lock options
+
+Some UI customization can be done via the `options` parameter when creating a `Lock` instance.
+
+
+### Theme options
+
+You can set custom theme properties like using a different logo, or changing the primary color.
+Just add a `theme` property with custom values.
```
Comment (flaviogranero) [inline articles/client-platforms/react/10-customizing-lock.md]: I think we could make it clear we're changing AuthService class. maybe with something like: Just add a `theme` option when initializing the auth0 Lock widget... 
Below you see our `AuthService` helper class updated to set a custom theme:

File: articles/client-platforms/react/10-customizing-lock.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### Results
+
+This is how it looks like using custom logo, color, and title:
+
+![Custom lock](/media/articles/reactjs/widget-custom-logo-color.png)
+
+
+<!-- CSS specification -->
+
+### 2. All done!
```
Comment (flaviogranero) [inline articles/client-platforms/react/10-customizing-lock.md:79]: what do think about adding numbers to all titles, keeping them at the same h2 level?

they could be:
1. Customize colors and image with theme options
2. Customize captions and labels with Language Dictionary option
3. Check the Results
4. All Done!

Comment (alemata) [inline articles/client-platforms/react/10-customizing-lock.md:79]: mm the idea is that there are two possible ways to customize: with options, and with css, so ideally would be:
1.Lock Options
- Theme
- Lang
- Check results

2.CSS
- Something
- Check results

3.All Done!

I am not sure what's the best...

Comment (flaviogranero) [inline articles/client-platforms/react/10-customizing-lock.md:79]: we have the "customizing with CSS" in this doc?

Comment (alemata) [inline articles/client-platforms/react/10-customizing-lock.md:79]: Not yet, they have to finish some Lock stuff first

**Comment by mpaktiti** [comment]
Please remove the pending label once the PR is ready for merge.

**Comment by flaviogranero** [comment]
@mpaktiti this is ready to be merged

**Pull request closed without merging**